pkg/ery/di/common_providers.go

```diff
@@ -0,0 +1,17 @@
+package di
+
+import (
+	"github.com/google/go-cloud/wire"
+
+	"github.com/srvc/ery/pkg/app/api"
+	"github.com/srvc/ery/pkg/app/dns"
+	"github.com/srvc/ery/pkg/ery"
+)
+
+func ProvideAPIConfig(cfg *ery.Config) *api.Config { return &cfg.API }
```

Comment:
**[golint]** <sub>reported by [reviewdog](https://github.com/haya14busa/reviewdog) :dog:</sub>
exported function ProvideAPIConfig should have comment or be unexported